assets/javascripts/discourse/components/discourse-post-event/dates.gjs

```diff
@@ -0,0 +1,84 @@
+import Component from "@glimmer/component";
```

Comment:
This one is a little bit tricky as most of it was computed in the initializer before

Reply:
There are two cases:
- local dates is enabled
- local dates is disabled